from PyQt5.uic import loadUi
from PyQt5.QtWidgets import QApplication


def on():
    fen.on_label.setVisible(True)
    fen.sc_mode_frame.setVisible(False)
    fen.input_1.clicked.connect(input1)
    fen.input_2.clicked.connect(input2)
    fen.input_3.clicked.connect(input3)
    fen.input_4.clicked.connect(input4)
    fen.input_5.clicked.connect(input5)
    fen.input_6.clicked.connect(input6)
    fen.input_7.clicked.connect(input7)
    fen.input_8.clicked.connect(input8)
    fen.input_9.clicked.connect(input9)
    fen.input_0.clicked.connect(input0)
    fen.par_op.clicked.connect(parop)
    fen.par_cl.clicked.connect(parcl)
    fen.input_comma.clicked.connect(comma)
    fen.add.clicked.connect(add)
    fen.minus.clicked.connect(minus)
    fen.multiply.clicked.connect(multiply)
    fen.divide.clicked.connect(divide)
    fen.equal.clicked.connect(equal)
    fen.back.clicked.connect(back)
    fen.actionScientific_Mode.triggered.connect(sc_mode)
    fen.actionNormal_Mode.triggered.connect(norm_mode)


app = QApplication([])
fen = loadUi("gui_calc.ui")
fen.show()
fen.resize(418, 355)
fen.sin.setVisible(False)
fen.cos.setVisible(False)
fen.tan.setVisible(False)
fen.asin.setVisible(False)
fen.acos.setVisible(False)
fen.atan.setVisible(False)
fen.sqrt.setVisible(False)
fen.pow.setVisible(False)
fen.abs.setVisible(False)
fen.e.setVisible(False)
fen.ln.setVisible(False)
fen.pi.setVisible(False)
fen.on_label.setVisible(False)
fen.on_c_button.clicked.connect(on)
app.exec_()
